pkg/dispers/aggregations/univariate.go

```diff
@@ -0,0 +1,39 @@
+package aggregations
+
+// Sum takes in input the data
+// an array pointing the variables on which apply the function
+// a map speciafying some parameters
+func Sum(data []map[string]interface{}, args map[string]interface{}) ([]float64, error) {
```

Comment:
It would be nice to have a better description of the expected input data format. Typically you have keywords like "key" and "weight", so a particular format is expected